docs/styling-the-navigator.md

```diff
@@ -79,7 +79,10 @@ this.props.navigator.setStyle({
   // Android only
   navigationBarColor: '#000000', // change the background color of the bottom native navigation bar.
   navBarTitleTextCentered: true, // default: false. centers the title.
-  topBarElevationShadowEnabled: false, // default: true. Disables TopBar elevation shadow on Lolipop and above
+  topBarElevationShadowEnabled: false, // (Android - default: true, iOS - default: false). Disables TopBar elevation shadow on Lolipop and above
+  shadowOpacity: .3, // iOS only - Works only when topBarElevationShadowEnabled: true
```

Comment:
Please add rename `shadowOpacity`, `shadowOffset` and `shadowRadius` to `topBarShadowOpacity`, `topBarShadowOffset` and `topBarShadowRadius` and move them to the `iOS only` section of this doc 👍